specification/app/resource-manager/Microsoft.App/preview/2022-06-01-preview/ManagedEnvironments.json

```diff
@@ -841,6 +893,34 @@
           "x-ms-secret": true
         }
       }
+    },
+    "WorkloadProfile": {
+      "description": "Description of a workload profile.",
```

Comment:
will this be an array ?

Reply:
And mismatch with file [ManagedEnvironments_CreateOrUpdate.json](https://github.com/Azure/azure-rest-api-specs/pull/19770/files#diff-6f12fcd602a58a7be78a0f0b9719c4b9aa35cb76966a2ca2dc12201129d941ba), line 22 which indicate an array via "["